packages/apps/ferretdb/templates/postgres.yaml

```diff
@@ -6,19 +6,18 @@ metadata:
 spec:
   instances: {{ .Values.replicas }}
   enableSuperuserAccess: true
-
   minSyncReplicas: {{ .Values.quorum.minSyncReplicas }}
   maxSyncReplicas: {{ .Values.quorum.maxSyncReplicas }}
 
   monitoring:
     enablePodMonitor: true
-
   storage:
     size: {{ required ".Values.size is required" .Values.size }}
     {{- with .Values.storageClass }}
     storageClass: {{ . }}
     {{- end }}
-
+  walStorage:
+    size: 1Gi
```

Comment:
But default value of `max_wal_size` is also 1GB. This means postgresql will hit out-of-space on first full WAL file that wasn't closed by the checkpoint that occurred before the file size limit.

Max WAL file size needs to be decreased for proper operation. I'd suggest 256MB. This will allow having at least two full WALs, and also preserve some room for replica startup operations.

Ex:

```
  postgresql:
    parameters:
      max_wal_size: "256MB"
```

This is applicable here and in other places where `walStorage.size` was introduced.